tests/modules/hapibd/main.nf

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { HAPIBD } from '../../../modules/hapibd/main.nf'
+
+workflow test_hapibd {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/hapibd/target.truth.vcf.gz", checkIfExists: true)
+    ]
+    map = file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/hapibd/target.map", checkIfExists: true)
```

Comment:
is this testdata file going to be moved outside of `delete_me` directory?

Reply:
Yeah, James suggested me to put the test input files in there :)